src/sentry/event_manager.py

```diff
@@ -2160,9 +2162,20 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects):
             ).select_related("group")
 
             new_grouphashes = set(group_hashes) - {hash.hash for hash in existing_grouphashes}
-            new_grouphashes_count = len(new_grouphashes)
 
             if new_grouphashes:
+                # temporary fix to limit group creation to grouphashes seen 3+ times in a 24-48 hour period
+                if settings.SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE:
+                    groups_to_create = new_grouphashes.copy()
```

Comment:
So I realised this this is actually kind of a rate limiting situation, but in reverse. We could potentially use our existing rate limiter with a different key that's specific to the group hash. https://github.com/getsentry/sentry/blob/5c57446306cfd4260c3646f2d0941a9c9fb56c54/src/sentry/event_manager.py#L2180-L2188

The quota would be 2/day. Once we are over the rate limit, then we can create the group. That saves us effort in provisioning a new redis etc, since we already have a cluster provisioned for our existing rate limiter.

Reply:
I looked into the `SlidingWindowRateLimiter` but didn't really love its options. Like we're setting up a static window size and granularity. I'd rather just keep it minimal (though the 24 hour TTL can go up to 48 hours) and simple, though agreed on the cluster provisioning

Reply:
Fair enough, regardless we can probably use the same redis cluster that the rate limiter uses